core/src/main/scala/org/apache/spark/sql/rapids/tool/qualification/QualificationAppInfo.scala

```diff
@@ -607,6 +612,7 @@ case class EstimatedSummaryInfo(
     recommendation: String,
     unsupportedExecs: String,
     unsupportedExprs: String,
+    var estimatedFrequency: Long,
```

Comment:
similar with all of these, why are they var's? Generally in Scala you don't want to use var, everything should be immutable unless absolutely necessary.  I didn't see them being set after the first time but maybe I missed it

Reply:
A comment that explains that `estimatedFrequency` is meant to be "number of jobs per month" would actually be helpful.